release/_github.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+function invoke_github_api {
+	local curl_response=$(\
+		curl \
+			"https://api.github.com/repos/kiwm/${1}" \
```

Comment:
@kiwm, can you please handle the first parameter as the full URL?

I mean, use `invoke_github_api_with_response "https://api.github.com/repos/kiwm/liferay-portal-ee/git/refs/heads/${original_branch_name}"`. I believe it'll make it clearer. :slightly_smiling_face: 

Additionally, can you please use the `liferay` user instead of yours?